build/Dockerfile.allinone

```diff
@@ -0,0 +1,39 @@
+FROM registry.access.redhat.com/ubi8/go-toolset:1.13.15 AS builder
+USER root
+WORKDIR /scratch
+COPY . .
+
+# Rather than echo the contents of terminal_check_unit.go as in the Makefile, we just copy it since the
+# file in sirupsen is essentially identical to what is being generated.  If that changes, so will this.
+RUN go mod vendor &&\
+    cp vendor/github.com/sirupsen/logrus/terminal_check_unix.go vendor/github.com/Sirupsen/logrus/terminal_check_unix.go
+
+RUN go build -o build/_output/bin/kfctl -gcflags all=-trimpath=/scratch -asmflags all=-trimpath=/scratch -mod=vendor github.com/kubeflow/kfctl/v3/cmd/manager
+
+# Add in the odh-manifests tarball
+RUN mkdir -p /opt/manifests &&\
+    tar -czf /opt/manifests/odh-manifests.tar.gz \
+        --exclude={./.*,*.md,Makefile,Dockerfile,Containerfile,LICENSE,OWNERS,tests} \
```

Comment:
I changed some things around, this exclude flag might not be quite right, checking ...